lib/help.js

```diff
@@ -30,6 +30,10 @@ Common Options:
   -l | --launch   Specifies the URL to launch during serve.
                     Possible values are host (default), local, or none.
                     Ex: skyux serve --launch local
+  -t | --template Specifies the base template to use when creating a SKY UX application.
```

Comment:
Just a heads up @blackbaud-johnly.  Once merged and released, we should add this to https://developer.blackbaud.com/skyux2/learn/reference/cli-commands